mixer/adapter/cloudwatch/cloudwatch.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY Type, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate $GOPATH/src/istio.io/istio/bin/mixer_codegen.sh -f mixer/adapter/cloudwatch/config/config.proto
+
+package cloudwatch
+
+import (
+	"context"
+
```

Comment:
style nit: this needs a formatting to clean up the `import` clause.

Reply:
make format doesn't correct it. What needs to change?

Reply:
ugh. i guess the PR to fix that still has not been committed. sorry.

easiest fix is just removing the blank line between `istio.io/istio/mixer/adapter/cloudwatch/config` and `istio.io/istio/mixer/pkg/adapter`